.github/workflows/release-automation.yml

```diff
@@ -0,0 +1,24 @@
+name: Update flyteidl version 
+
+on:
+  schedule:
+    # TODO: Need to discuss
```

Comment:
Shouldn't this happen on release creation?

Reply:
If you use something like this: https://github.com/marketplace/actions/dependencies-autoupdate it will make sure it doesn't recreate a PR but rather just update the same branch with the latest.... so it'll keep it as a single PR until we merge it then start creating a new one for the next update... etc. would that help?